recipes/pytdc/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "pytdc" %}
+{% set version = "0.0.5" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/PyTDC-{{ version }}.tar.gz
+  sha256: a11b9688809b9ed4b54d863a813b412f0ac28eeb7293ca56c8fb4cf7fdd52189
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - setuptools >=38.6.0
+  run:
+    - python >=3.6
+    - fuzzywuzzy
+    - numpy
+    - pandas >=1.1.4
+    - scikit-learn >=0.21.3
+    - tqdm
+    - scipy
+    - networkx
+    - rdkit >=2020.03.6
+    - requests
+
+test:
+  imports:
+    - tdc
+    - tdc.generation
+    - tdc.multi_pred
+    - tdc.single_pred
+  requires:
```

Comment:
Unless you're running `pip check` as a test, I don't think you need the `requires` section

Reply:
Test files are not included in the pypi source package. Will delete.